airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -20,15 +20,20 @@
 import json
 import warnings
 from datetime import timedelta
-from typing import TYPE_CHECKING, Sequence
+from typing import TYPE_CHECKING, Any, Sequence
 
+from astroid.decorators import cachedproperty
```

Comment:
@vandonr-amz note this raises warning in the CI:
`airflow/providers/amazon/aws/operators/rds.py:78: [W0513(warning), ] cachedproperty has been deprecated and will be removed in astroid 3.0 for Python 3.8+. Use functools.cached_property instead.`

Reply:
@eladkal 

Fix here: https://github.com/apache/airflow/pull/33133

Reply:
Raphael just left for a vacation :)

Reply:
thanks, I guess I was a bit too trigger-happy with auto-resolution of imports